VK_App/VK_App/Model/News/NewsFeed.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Items.swift
+//  VK_App
+//
+//  Created by Анастасия Козлова on 06.12.2022.
+//
+
+import Foundation
+
+/// Новость
+final class NewsFeed: Decodable {
+    
+    var id: Int
+    var sourceID: Int
```

Comment:
```suggestion
    var id: Int
    var sourceID: Int
```
а для этих свойств описание?